byzcoin/bcadmin/cmd_db.go

```diff
@@ -43,17 +43,22 @@ func dbStatus(c *cli.Context) error {
 
 // dbCatchup uses the live byzcoin chain to update to the latest blocks
 func dbCatchup(c *cli.Context) error {
-	if c.NArg() < 3 {
+	if c.NArg() < 1 {
 		return xerrors.New("please give the following arguments: " +
-			"conode.db byzCoinID url")
+			"conode.db [byzCoinID [url]]")
 	}
 	fb, err := newFetchBlocks(c)
 	if err != nil {
 		return xerrors.Errorf("couldn't create fetchBlock: %+v", err)
 	}
-	err = fb.addURL(c.Args().Get(2))
```

Comment:
does nothing when only two args?

Reply:
The two args case is handled in `newFetchBlocks`.